pkg/components/tablet_node.go

```diff
@@ -205,6 +205,23 @@ func (tn *TabletNode) initBundles(ctx context.Context) (ComponentStatus, error)
 		}
 	}
 
+	for _, bundle := range []string{DefaultBundle, SysBundle} {
+		bootstrap := tn.getBundleBootstrap(bundle)
+		nodeTagFilter := ""
```

Comment:
this is unused one and ends up in a log, which is misleading, because different value is being set.